internal/config/config.go

```diff
@@ -150,6 +158,8 @@ func setNonCliDefaultValues(v *viper.Viper) {
 	v.SetDefault("log.level", "")
 	v.SetDefault("log.file", "")
 	v.SetDefault("log.structured", false)
+	v.SetDefault("registration.always-use-fallback-register-id", false)
```

Comment:
do these not also need to be defined in the RegistrationOptions struct? if they aren't designed to be configurable by the user not sure this is the right place

Reply:
oops, good catch. These are leftovers that should have been removed. I missed that. Will fix.

My thinking is that we have no defaults for registration-id, integration-name and integration-description.